test/cmd/builds.sh

```diff
@@ -12,16 +12,64 @@ os::log::install_errexit
 url=":${API_PORT:-8443}"
 project="$(oc project -q)"
 
+# expect_build_output_spec runs `oc new-build` with given arguments and expects
+# a build config with a given name to be created with a given build output spec
+function os::cmd::builds::expect_build_output_spec() {
```

Comment:
Also, small nit - this function should live in `hack/builds.sh` once https://github.com/openshift/origin/issues/5844 is sorted out, and maybe in `hack/cmd_util.sh` or `hack/util.sh` for now, not here.